noir/Earthfile

```diff
@@ -1,6 +1,9 @@
 VERSION 0.8
 
 nargo:
+    LOCALLY
+    LET git_commit = $(git rev-parse HEAD)
```

Comment:
@ludamad thoughts on this change? I refactored the test docker image to reuse the already built nargo instead of building from scratch ([see script](https://github.com/AztecProtocol/aztec-packages/blob/fbd78fdc53071f3548971dfb4832a440512f4687/noir/scripts/test_js_packages.sh#L13)) and that requires the source hashes to be the same. Using the `shasum` trick wouldn't work because the source copied into the build containers would be different.

Reply:
This causes a rebuild for every single git hash! This defeats caching unfortunately

Reply:
There's a reason I hacked a content hash